dashboard/test/controllers/api/v1/pd/international_optins_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class Api::V1::Pd::RegionalPartnerContactsControllerTest < ::ActionController::TestCase
+  SAMPLE_FORM_DATA = {
+    firstName: 'First',
+    firstNamePreferred: 'Preferred',
+    lastName: 'Last',
+    email: 'foo@bar.com',
+    emailAlternate: 'footoo@bar.com',
+    gender: 'Prefer not to answer',
+    schoolName: 'School Name',
+    schoolCity: 'School City',
+    schoolCountry: 'School Country',
+    ages: ['19+ years old'],
+    subjects: ['ICT'],
+    resources: ['Kodable'],
+    robotics: ['LEGO Education'],
+    workshopOrganizer: 'Workshop Organizer',
+    workshopFacilitator: 'Workshop Facilitator',
+    workshopCourse: 'Workshop Course',
+    optIn: 'Yes'
+  }
+
+  test 'create creates a new international optin' do
+    assert_creates Pd::InternationalOptin do
```

Comment:
Not sure why this fails...

Reply:
Perhaps the API call is failing for some reason? I'd suggest moving the `assert_response` inside the `assert_creates` so we first assert the successful response and then assert that it created a new model instance.

Also does `assert_response :created` (201) work? I think we return 200 `:success`. In any case, I suspect we're failing to create and returning a 400 or other error code. See https://www.rubydoc.info/gems/rack/Rack/Utils for the list of codes.